pagopa/io-app
# chore(IT Wallet): [SIW-952] Add CGN card in new wallet section

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5625

## Short description
This PR adds the CGN card in the new wallet section

## List of changes proposed in this pull request
- Added `CgnCard`and `CgnWalletCard` component
- Added action dispatch in the new wallet section

## How to test
Enable the **New wallet section** FF within the **Profile** section in the app, check that the CGN card is correctly displayed in the new wallet. 
Note: CGN card can only be requested within the old wallet section.

## Preview
<img src="https://github.com/pagopa/io-app/assets/6160324/a6e1b0e7-b8c9-47e4-a67c-cf081356b44d" width="200" />
